crates/ruff_db/src/files.rs

```diff
@@ -308,6 +308,9 @@ pub struct File {
     count: Count<File>,
 }
 
+// The Salsa heap is tracked separately.
+impl get_size2::GetSize for File {}
```

Comment:
Can't we derive the implementation for `GetSize` instead, considering that it will be derived on the final struct (that only wraps the id)?

Reply:
Unfortunately `salsa::Id` does not implement `GetSize`, and we have no way of putting an `ignore` attribute on that field.